### ISSUE TITLE
Kolab Roundcube plugins mirror is very outdated

### ISSUE THREAD

**Issue by CodeShakingSheep** [roundcube/roundcubemail#9504]
### Prerequisites

- [X] I have [searched](https://github.com/roundcube/roundcubemail/issues?q=is%3Aissue) for duplicate or closed issues
- [X] I can recreate the issue with all plugins disabled

### Describe the issue

I just wanted to install the `kolab/calendar` plugin and found that it's really outdated (last commit on master from September 2021) and latest release 3.5.7 is from May 2021. See https://plugins.roundcube.net/#/packages/kolab/calendar and https://github.com/kolab-roundcube-plugins-mirror/calendar. However, when I go to kolab's git there is recent activity (from this year) and the latest release is 3.5.11 from January 2022.

The same applies to `kolab/tasklist` plugin. See https://plugins.roundcube.net/#/packages/kolab/tasklist, https://github.com/kolab-roundcube-plugins-mirror/tasklist and https://git.kolab.org/diffusion/RPK/browse/master/plugins/tasklist/

I'm not sure if this is the right place to report this but in the GitHub kolab repositories issues were disabled. I'd appreciate the GitHub mirrors being updated to be able to provide the latest plugin packages on packagist.

### What browser(s) are you seeing the problem on?

Chrome

### What version of PHP are you using?

v8.1

### What version of Roundcube are you using?

1.6.7

**Comment by skmedia1** [comment]
You're absolutely right in pointing out this issue. It's clear that the Kolab plugin repositories on GitHub are outdated compared to the activity on Kolab's official Git. 

Unfortunately, this isn't an uncommon situation with open-source projects where mirrors or package managers like Packagist don't get updated regularly. 

As for the broader usage of Roundcube, it's true that it is primarily seen as a simple webmail interface, often in academic or smaller server environments, and not typically in larger enterprise setups. Many users find that the calendar and tasklist functionalities lack some of the advanced features needed for more intensive use. However, adapting or transforming the plugins, as we've done with certain applications, could indeed be a possible solution for your specific needs.

**Comment by alecpl** [comment]
The mirrors have been update with version 3.6.0.

**Comment by CodeShakingSheep** [comment]
Hi @alecpl ,
That's awesome, thank you! I want to also specifically ask about kolab_2fa. There doesn't seem to be a mirror of it on GitHub and the latest version in the RC plugin repository is from 2021, see https://plugins.roundcube.net/#/packages/kolab/kolab_2fa while here https://git.kolab.org/diffusion/RPK/browse/master/plugins/kolab_2fa/ the latest version is from beginning of this year. Any way to update the plugin in RC plugin repository?
